tools/topology/sof/pipe-eq-capture-16khz.m4

```diff
@@ -0,0 +1,108 @@
+# Capture EQ Pipeline and PCM, 16 kHz
```

Comment:
is the reason this is fixed to 16kHz simply because that is what the tuning parameters were set for?

Reply:
I think it's Intel FW legacy to setup the DAI for 48 kHz and 16 kHz configurations and we are following it. The 16 kHz is a typical sample rate for automatic speech recognition. Though in SOF case it could be a different rate to match needs.